graylog2-web-interface/src/components/event-notifications/event-notification-form/EventNotificationForm.jsx

```diff
@@ -116,6 +129,7 @@ class EventNotificationForm extends React.Component {
               <ButtonToolbar>
                 <Button bsStyle="primary" type="submit">{action === 'create' ? 'Create' : 'Update'}</Button>
                 <Button onClick={onCancel}>Cancel</Button>
+                <Button bsStyle="info" disabled={testButtonDisabled} onClick={() => onTest(notification)}> {testButtonText} </Button>
```

Comment:
The test button looks a bit misplaced right next to the other buttons. What about making it bigger and align it to the right so that the user sees it's not related to a CRUD option? Also a small text would help to make sure the user knows what this button does. Something like `Triggers this notification with dummy data for an end-to-end test`